example.ts

```diff
@@ -0,0 +1,37 @@
+import { IAssignmentLogger } from '@eppo/js-client-sdk-common';
+
+import { init } from './src/index';
+
+const assignmentLogger: IAssignmentLogger = {
+  logAssignment(assignment) {
+    console.log('assignement', assignment);
+  },
+};
+
+async function main() {
+  const eppoClient = await init({
+    apiKey: 'O8xGu8cegyGTWYq8Vc02QMmhez-fz8mjL4ajY247kro',
```

Comment:
The way the Android example app works is it creates a simple app with two inputs: flag key and subject key, and a button to request an assignment. This, combined with the ability to provide the API key via a `local.properties` file (we could do environment) means users can run the example apps against their Eppo keys and flags. I think this is the pattern we want to do for other SDK example apps.